hamuli/Teamwork
# #168345564 implement sign in feature

- this feature allow the user to sign in
[finishes #168345564]

#### What does this PR do?
- this feature allow us to sign in .
#### Description of Task to be completed?
- before sharing , user have to  get access to the system by sign in then he can post and comment . 
#### How should this be manually tested?
- the password enter by the user should be the same enter when he created his account.
- and the username should be the same with the one he create his account.
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/168345564
#### Screenshots
